doc/nrf/links.txt

```diff
@@ -482,6 +482,18 @@
 .. _`Edge Impulse's data forwarder`: https://docs.edgeimpulse.com/docs/cli-data-forwarder
 .. _`Edge Impulse CLI installation guide`: https://docs.edgeimpulse.com/docs/cli-installation
 
+.. ### Source: pelion.com
+
+.. _`Pelion website`: https://pelion.com/
+.. _`Pelion documentation`: https://developer.pelion.com/docs/
+.. _`Pelion Device Management`: https://developer.pelion.com/docs/device-management/current/welcome/index.html
+.. _`Provisioning devices for Pelion Device Management`: https://developer.pelion.com/docs/device-management/current/provisioning-process/index.html
+.. _`Provisioning development devices`: https://developer.pelion.com/docs/device-management/current/provisioning-process/provisioning-development-devices.html
+.. _`Firmware-over-the-air (FOTA) updates`: https://developer.pelion.com/docs/device-management/current/updating-firmware/index.html
```

Comment:
@greg-fer @melwee01 
We are using the old update client for firmware updates on Zephyr so the information should be aligned with 4.7: 
https://developer.pelion.com/docs/device-management/v4.7/updating-firmware/index.html

Reply:
changed